docs/plugins/cursor/rules-use.md

```
@@ -160,6 +170,7 @@ Use `@filename` syntax to include external file content:
 @templates/component-template.tsx
 
 # Additional Guidelines
+
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/2/3

(MD029, ol-prefix)